addons/bus/static/src/debug/bus_logs_service.js

```diff
@@ -0,0 +1,33 @@
+import { reactive } from "@odoo/owl";
+
+import { registry } from "@web/core/registry";
+
+export const busLogsService = {
+    dependencies: ["bus_service", "multi_tab"],
+    /**
+     * @param {import("@web/env").OdooEnv}
+     * @param {Partial<import("services").Services>} services
+     */
+    start(env, { bus_service, multi_tab }) {
+        const state = reactive({
+            enabled: multi_tab.getSharedValue("bus_log_menu.enabled", false),
+            toggleLogging() {
+                state.enabled = !state.enabled;
+                bus_service.setLoggingEnabled(state.enabled);
+                multi_tab.setSharedValue("bus_log_menu.enabled", state.enabled);
+            },
+        });
+        multi_tab.bus.addEventListener("shared_value_updated", ({ detail }) => {
+            if (detail.key === "bus_log_menu.enabled") {
+                state.enabled = detail.newValue;
+            }
+        });
+        bus_service.setLoggingEnabled(state.enabled);
+        if (state.enabled) {
+            console.log("Bus logging is enabled. You can disable it from the debugging menu.");
+        }
+        return state;
```

Comment:
Would be nice to suggest a way to do it directly in dev tools, e.g. with `odoo.stopBusLogging()`.